src/Client/LightningStore.php

```diff
@@ -8,6 +8,7 @@
 use BTCPayServer\Result\LightningInvoice;
 use BTCPayServer\Result\LightningNode;
 use BTCPayServer\Result\LightningPayment;
+use BTCPayServer\Util\PreciseNumber;
```

Comment:
nitpick: can be removed, not used anymore

Reply:
nice catch, fixed.